tools/codegenerator/example/native/ModuleXbmc.h

```diff
@@ -1 +1 @@
-void log(const char* msg, int level = LOGNOTICE);
+void log(const char* msg, int level = LOGDEBUG);
```

Comment:
This is an example project for the codegenerator tool. Nothing to do with the other `log`